python/fusion_engine_client/analysis/data_loader.py

```diff
@@ -282,7 +287,7 @@ def _read(self,
               return_numpy: bool = False, keep_messages: bool = False, remove_nan_times: bool = True,
               time_align: TimeAlignmentMode = TimeAlignmentMode.NONE,
               aligned_message_types: Union[list, tuple, set] = None,
-              quiet: bool = False) \
+              quiet: bool = False, source_ids: Optional[List[int]] = None) \
```

Comment:
This parameter needs to be documented above. I already noted this, not sure where that comment went, but this should be moved up above so it is defined with the other filtering arguments (message types, time range), and the documentation order should match.